pyteal/compiler/optimizer/optimizer.py

```diff
@@ -87,13 +121,15 @@ def _apply_slot_to_stack(
     _remove_extraneous_slot_access(start, slots_to_remove)
 
 
-def apply_global_optimizations(start: TealBlock, options: OptimizeOptions) -> TealBlock:
+def apply_global_optimizations(
```

Comment:
This is a breaking change, but I'm comfortable with it because `apply_global_optimization` is internal